pkg/frr/frr.go

```diff
@@ -437,7 +430,7 @@ func setUpVrf(vrf *infradb.Vrf) (string, bool) {
 
 // checkFrrResult checks the vrf result
 func checkFrrResult(cp string, show bool) bool {
-	return ((show && reflect.ValueOf(cp).IsZero()) || strings.Contains(cp, "warning") || strings.Contains(cp, "unknown") || strings.Contains(cp, "Unknown") || strings.Contains(cp, "Warning") || strings.Contains(cp, "Ambiguous") || strings.Contains(cp, "specified does not exist"))
+	return ((show && reflect.ValueOf(cp).IsZero()) || strings.Contains(cp, "warning") || strings.Contains(cp, "unknown") || strings.Contains(cp, "Unknown") || strings.Contains(cp, "Warning") || strings.Contains(cp, "Ambiguous") || strings.Contains(cp, "specified does not exist") || strings.Contains(cp, "Error"))
```

Comment:
reflect.ValueOf(cp).IsZero()) -> cp == ""

what if cp contains `waRning`? Will you add a new comparison then? Will it be better to lower case the string and only then check what itcontains?

Reply:
Done

Reply:
Changed all local vars names in frr.go and replaced all reflect.valueof IsZero with simple comparison.